src/core/Target_file.ml

```diff
@@ -0,0 +1,19 @@
+type provided_target_language_info =
```

Comment:
a type with _info is not great.

Reply:
We're trying to move away from using the term "language" because it's ambiguous and restrictive in the context of semgrep rules. Please prefer "selector" or "target selection", etc. the goal being to select files however we like using a combination of filters e.g. `image and not pdf`.